localhost.crt

```diff
@@ -0,0 +1,21 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Could we store these in a subfolder somewhere -- the more files we add to the root directory, the more it pushes down the README, you know? Maybe in `config`?

Thank you!!!!